org.gnome.Contacts.json

```diff
@@ -0,0 +1,215 @@
+                "--disable-Werror",
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/libgdata",
+                    "tag": "0.17.9",
+                    "commit": "60f2269178d7a6449e2ffdf7fdf82d96a5aff0d4"
+                }
+            ]
+        },
+        {
+            "name": "evolution-data-server",
+            "cleanup": [
+                "/lib/cmake",
+                "/lib/evolution-data-server/*-backends",
+                "/libexec",
+                "/share/dbus-1/services"
+            ],
+            "config-opts": [
+                "-DENABLE_GTK=ON",
+                "-DENABLE_GOA=ON",
+                "-DENABLE_UOA=OFF",
+                "-DENABLE_GOOGLE_AUTH=ON",
+                "-DENABLE_GOOGLE=ON",
+                "-DENABLE_WITH_PHONENUMBER=ON",
+                "-DENABLE_VALA_BINDINGS=ON",
+                "-DENABLE_WEATHER=OFF",
+                "-DWITH_OPENLDAP=OFF",
+                "-DENABLE_INTROSPECTION=ON",
+                "-DENABLE_INSTALLED_TESTS=OFF",
+                "-DENABLE_GTK_DOC=OFF",
+                "-DENABLE_EXAMPLES=OFF"
+            ],
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/evolution-data-server",
+                    "tag": "3.28.0",
+                    "commit": "7f6c16388216840f6bbda0d98383d5c008bc8d45"
+                }
+            ]
+        },
+        {
+            "name": "folks",
+            "cleanup": [ "/bin", "/share/GConf" ],
+            "config-opts": [ "--disable-fatal-warnings" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/folks",
+                    "tag": "0.11.4",
+                    "commit": "9d29ec2c5d6763b079f56940d5aaf4c13b0b3510"
+                }
+            ]
+        },
+        {
+            "name": "gnome-desktop",
+            "config-opts": ["--disable-debug-tools", "--disable-udev"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/gnome-desktop",
+                    "tag": "3.28.0",
+                    "commit": "5e30118786eb767001fc8bfc8d130127b9538297"
+                }
+            ]
+        },
+        {
+            "name": "cheese",
+            "config-opts": [ "--disable-man", "--disable-lcov" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/cheese",
+                    "tag": "3.28.0",
+                    "commit": "355c121cd032837cad1bd54e1a560afc0712acba"
+                }
+            ]
+        },
+        {
+            "name": "gnome-contacts",
+            "buildsystem": "meson",
```

Comment:
Add config-opts for `--buildtype=release` (or debugoptimized if you prefer)

Reply:
Added!